external/netstandard/netstandard.depproj

```diff
@@ -23,6 +30,27 @@
       </Reference>
     </ItemGroup>
   </Target>
+
+  <Target Name="CopyToAdditionalOutputDirectories" 
```

Comment:
It needs to be done better and put into binplace.targets.  I didn't spend the time to solve that problem so I didn't centralize it.